tests/core/tools/manual/escapecss.md

```diff
@@ -0,0 +1,11 @@
+@bender-tags: 4.16.2, bug, 681
+
+1. Select the table cell in IE and open developer console.
+
+   ## Expected
```

Comment:
Our [code style](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#31-manual-tests) define `Expected` and `Unexpected` as bolded values - not headers.